spec/support/equipment_model_helper.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+module EquipmentModelHelper
```

Comment:
Oh ok got it. So we'd ultimately just be including one huge `ModelHelpers` module that maybe has sub-modules for each model (e.g. `ModelHelpers::EquipmentModelHelper`, etc)?